internal/interconnections/retrieve.go

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2018 Jasmin Gacic <jasmin@stackpointcloud.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package interconnections
+
+import (
+	"context"
+	"fmt"
+
+	metal "github.com/equinix-labs/metal-go/metal/v1"
+	"github.com/spf13/cobra"
+)
+
+func (c *Client) Retrieve() *cobra.Command {
+	// metrosCmd represents the metros command
+	var projectID, organizationID string
+	retrieveInterconnectionsCmd := &cobra.Command{
+		Use:     "get",
+		Aliases: []string{"list"},
+		Short:   "Retrieves a list of interconnections.",
+		Long:    "Retrieves a list of interconnections available to the default Organizations and by project ID",
+		Example: `  # Lists interconnections available to the default OrganizationID OR by project ID:	
+  metal interconnections get`,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			var request metal.ApiOrganizationListInterconnectionsRequest
+
+			if projectID != "" {
+				request = c.Service.OrganizationListInterconnections(context.Background(), projectID)
```

Comment:
It looks like there are pagination attributes for this endpoint; we should add a pager here to make sure we load everything.

Reply:
Also it looks like this line is calling the wrong endpoint; should be calling the project interconnections endpoint.